ql/src/test/results/clientpositive/llap/insert_values_orig_table_use_metadata.q.out

```diff
@@ -108,7 +108,7 @@ Table Parameters:
 	numFiles            	0                   
 	numRows             	0                   
 	rawDataSize         	0                   
-	totalSize           	#Masked#
+	totalSize           	#Masked#                   
```

Comment:
Need to regenerate this out once remove the `totalSize`.

Reply:
Done, thanks